### DISCUSSION TITLE
How we get adsense approval .

### DISCUSSION THREAD

**Discussion by Gouravyadavyt** [NXTS-Developers/MOFHY-Lite#71] [Q&A] [answered]
My dear friend ,
I try to get adsense account but the refuses my site and give me policy violence .
Https://easyteachhost.tk

**Comment by mahtab2003** [comment]
I don't know

**Comment by OverdueWeevil2** [comment, marked as answer]
Maybe this can help:
https://support.google.com/adsense/answer/7299563

**Comment by jaikt** [comment]
Free domains are against Adsense policy. You just need to have the homepage of your website unique. I realised this on my website: https://royalityfree.com

**Reply by Gouravyadavyt** [reply]
Your are really help full can you do two things for use .
tell what hosting you use and what theme .
Is it free

**Reply by jaikt** [reply]
I am using a custom theme (not free) and hosting on hostgator. The client area is kinda custom coded but the hosting functions are inspired by mofhy.

**Reply by mahtab2003** [reply]
Hmm...
